browser/src/slideshow/engine/AnimationFactory.ts

```diff
@@ -347,16 +519,22 @@ function createShapeTransition(
 			return null;
 
 		case TransitionClass.ClipPoligon: {
-			const aParametricPolyPolygon = createClipPolyPolygon(
-				eTransitionType,
-				eTransitionSubType,
-			);
-			const aClippingAnimation = new ClippingAnimation(
-				aParametricPolyPolygon,
-				aTransitionInfo,
-				bDirectionForward,
-				bModeIn,
+			const aClippingAnimation = new TransitionFilterAnimation(
+				aAnimatedTransitionFilterNode.getId(),
+				transitionFilterInfo,
+				aAnimatedElement,
 			);
+
+			// const aParametricPolyPolygon = createClipPolyPolygon(
+			// 	eTransitionType,
+			// 	eTransitionSubType,
+			// );
+			// const aClippingAnimation = new ClippingAnimation(
+			// 	aParametricPolyPolygon,
```

Comment:
dead comment? or for debbuging later?

Reply:
I'll remove when work on transition filter is over.